contracts/swaps/SwapsUser.sol

```diff
@@ -171,7 +172,8 @@ contract SwapsUser is State, SwapsEvents, FeesHelper {
 				_payTradingFee(
 					addrs[4], /// user
 					loanId, /// loanId,
-					addrs[1], /// destToken
+					addrs[1], /// destToken (feeToken)
```

Comment:
Is it correct that `specialRebates[sourceToken][destToken]` = `specialRebates[destToken][sourceToken]` since you pass `addrs[0]` as destination and `addrs[1]` as source?

Reply:
specialRebates[sourceToken][destToken] != specialRebates[destToken][sourceToken]
But can refer to the previous version where #1 _payTradingFee use address[0] as feeToken
https://github.com/DistributedCollective/Sovryn-smart-contracts/blob/development/contracts/swaps/SwapsUser.sol#L134

Meanwhile the #2 _payTradingFee use address[1] as feeToken
https://github.com/DistributedCollective/Sovryn-smart-contracts/blob/development/contracts/swaps/SwapsUser.sol#L174